roles/ansible-test/tasks/init_collection.yaml

```diff
@@ -14,15 +14,26 @@
   shell: "cat galaxy.yml | {{ ansible_test_venv_path }}/bin/yq -y .name | tail -n +1 | head -1"
   register: _collection_name
 
-- name: Setup location of project
+- name: Setup location of project for integration tests
+  vars:
+    _test_type: "integration"
   set_fact:
     _test_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}"
-    _test_cfg_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}/tests/integration/{{ ansible_test_test_command }}.cfg"
+    _test_cfg_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}/tests/{{ _test_type }}/{{ ansible_test_test_command }}.cfg"
+  when: ansible_test_test_command == 'network-integration'
+
+- name: Setup location of project for unit tests
+  vars:
+    _test_type: "unit"
+  set_fact:
+    _test_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}"
+    _test_cfg_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}/tests/{{ _test_type }}/{{ ansible_test_test_command }}.cfg"
+  when: ansible_test_test_command == 'network-unit'
```

Comment:
network-unit doesn't appear to be the right sub command, lets change it to just be 'units'